### PR TITLE
{bio}[system/system] Coot v0.9.8.92 w/ binary Linux x86_64 scientific linux 7.6 python gtk2

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Review by migueldiascosta — DISMISSED** [review]
lgtm

File: easybuild/easyconfigs/c/Coot/Coot-0.9.8.92-binary-Linux-x86_64-scientific-linux-7.6-python-gtk2.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+toolchain = SYSTEM
+
+source_urls = ['http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot/binaries/release']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['3babef48f5cea17c702347e41c51e7492192c685ef1c7d8b796569eb9f80d8d9']
+
+sanity_check_paths = {
+    'files': ["bin/coot"],
+    'dirs': ["bin"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/Coot/Coot-0.9.8.92-binary-Linux-x86_64-scientific-linux-7.6-python-gtk2.eb:25]: nitpicking but it doesn't make sense to check for the `bin` dir when we're already checking for `bin/coot`, maybe change this to `lib`?

Comment (verdurin) [reply easybuild/easyconfigs/c/Coot/Coot-0.9.8.92-binary-Linux-x86_64-scientific-linux-7.6-python-gtk2.eb]: Yes, made that change now. I just inherited what was there before.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @verdurin!

**Pull request merged**